lib/model/profile_model.dart

```diff
@@ -46,26 +46,32 @@ class ProfileModel extends DataModel<ProfileModel> implements StorageData {
             setOwnerRequired: false);
 
   // implement copyWith Method
-  ProfileModel copyWith(
-      {String? name,
-      String? email,
-      int? color,
-      String? nickname,
-      String? slogan,
-      String? introduction,
-      List<ProfileTag>? tags,
-      io.File? photo,
-      List<String>? associateEntityId}) {
+  ProfileModel copyWith({
+    String? id,
+    String? databasePath,
+    bool? storageRequired,
+    bool? setOwnerRequired,
```

Comment:
the id, databasePath, storageRequired, setOwnerRequired is not allow to be change